gcloud/credentials.py

```diff
@@ -72,8 +73,34 @@ def get_credentials():
     return client.GoogleCredentials.get_application_default()
 
 
+def get_for_service_account_json(private_key_path, scope=None):
```

Comment:
Exactly -- `private_key` made me think I should send in a .p12... not a "json credentials file". Maybe rename to `json_credentials_path` ?

(Again, just a nit...)